pkg/config/runtime.go

```diff
@@ -87,7 +88,7 @@ func (r *Runtime) resolveRules(bag attribute.Bag, aspectSet AspectSet, rules []*
 	var lerr error
 
 	for _, rule := range rules {
-		if glog.V(2) {
+		if glog.V(3) {
```

Comment:
curious: why do we use this form here, and the inline form below?  is rule that much of a pain to stringify in comparison?